uwb-beacon-firmware/src/ranging_thread.c

```diff
@@ -154,6 +170,13 @@ static void ranging_thread(void* p)
             uwb_send_anchor_position(&handler, x, y, z, frame);
         }
 
+        if (flags & EVENT_DATA_PACKET_READY) {
+            dwt_forcetrxoff();
+            uwb_send_data_packet(&handler, data_packet_dst_mac, data_packet, data_packet_size, frame);
+
+            chBSemSignal(&data_packet_sent);
+        }
```

Comment:
so we allow a beacon to run both as a positioning beacon and a communication relay?

Reply:
For now yes, as it seems to work OK. If we realize it is an issue, we can always change this, what do you think ?

Reply:
sounds good, ship it!